src/ThrowablePhp5.php

```diff
@@ -0,0 +1,10 @@
+<?php
+
+namespace Psr\SimpleCache;
+
+/**
+ * Interface for PHP 5 backward compatibility.
+ */
+interface ThrowablePhp5
```

Comment:
you are not solving the problem here, `ThrowablePhp5` is still just an interface, not `\Exception` subclass.

this should probably be `class ThrowablePhp5 extends \Exception`

Reply:
`CacheException` is an interface, so can't extends a class.
That's solve the problem for PHP 7.

Reply:
@glensc For PHP 5 there is no way to solve the problem as there is no interface for `Exception`s. There shouldn't be any implementations as the repository only contains interfaces. The `ThrowablePhp5` is there only for BC.